weeklyupdates/templates/common.xhtml

```diff
@@ -68,8 +73,11 @@ import json
         </div>
       </py:when>
       <py:otherwise>
-        <h4 py:if="showuser" class="postuser"><a href="${genlink('user', post.userid)}">
-          <img class="avatar" src="https://avatars.io/email/${post.userid}" />${post.userid}</a>
+			<h4 py:if="showuser" class="postuser"><a href="${genlink('user', post.userid)}">
+          <py:if test="show_avatar">
+          <img class="avatar" src="https://avatars.io/email/${post.userid}" />
```

Comment:
This should be &lt;img py:if="show_avatar" class="avatar" src=..."> to avoid the extra py:if verbosity.